.github/workflows/build.yml

```diff
@@ -29,6 +29,7 @@ jobs:
         hatch run lint-check
     - name: Test
       run: |
+        hatch run test
```

Comment:
Since the two adjacent lines here are the same,
https://github.com/pinellolab/DNA-Diffusion/blob/a55002dfdf21633cfb8b1b4a1f34d39945bf3009/pyproject.toml#L141
this should be redundant (i.e. the new functionality would essentially run the tests twice).